eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/AbstractBiMap.java

```diff
@@ -371,7 +371,7 @@ public void forEachKeyValue(Procedure2<? super K, ? super V> procedure)
     @Override
     public void each(Procedure<? super V> procedure)
     {
-        this.getInverse().forEachKey(procedure);
+        this.getDelegate().forEachValue(procedure);
     }
```

Comment:
I thought this was handled separately - I don't remember where that commit lives. There's a second method that needs this change, `each()` and it's a bug-fix so I think it makes sense to fix in an earlier commit with dedicated tests.